ui/imports/shared/panels/chat/ChatReplyPanel.qml

```diff
@@ -66,7 +66,7 @@ Loader {
             Shape {
                 id: replyCorner
                 anchors.left: parent.left
-                anchors.leftMargin: 20 - 1
+                anchors.leftMargin: 24 - 1
```

Comment:
Hmm....

Reply:
Consequence of changing the size of the user icon. I agree, all those magic numbers should be refactored to some constants...